tests/Fixer/Import/FullyQualifiedStrictTypesFixerTest.php

```diff
@@ -36,6 +36,19 @@ public function testCodeWithReturnTypes($expected, $input = null)
         $this->doTest($expected, $input);
     }
 
+    /**
+     * @requires PHP 7.1
+     *
+     * @dataProvider provideCodeWithReturnTypesCasesWithNullableCases
+     *
+     * @param mixed      $expected
```

Comment:
~~Can we have `string` here, and in the line below, instead of `mixed`?~~
Actually, can we have `string` instead of `mixed` in the whole file?

Reply:
I think yes.

Reply:
done